frappe/database.py

```diff
@@ -520,7 +525,13 @@ def get_list(self, *args, **kwargs):
 		return frappe.get_list(*args, **kwargs)
 
 	def get_single_value(self, doctype, fieldname, cache=False):
-		"""Get property of Single DocType. Cache locally by default"""
+		"""Get property of Single DocType. Cache locally by default
+		:param doctype: DocType name.
+		:param fieldname: Column name.
+		Example:
```

Comment:
leave enough whitespace for markdown formatting

Also add a tab for code formatting

```
:param doctype: DocType name.
:param fieldname: Column name.

Example:

    company = frappe.db.get_single_value('Global Defaults', 'default_company')
   
```